src/Workspaces/VisualBasic/Portable/Formatting/DefaultOperationProvider.vb

```diff
@@ -94,27 +94,29 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Formatting
             Return Nothing
         End Function
 
-        Private Function IsSingleLineIfOrElseClauseSyntax(node As SyntaxNode) As Boolean
+        Private Shared Function IsSingleLineIfOrElseClauseSyntax(node As SyntaxNode) As Boolean
             Return TypeOf node Is SingleLineIfStatementSyntax OrElse TypeOf node Is SingleLineElseClauseSyntax
         End Function
 
-        Private Function ColonOrLineContinuationTrivia(trivia As SyntaxTrivia) As Boolean
+        Private Shared Function ColonOrLineContinuationTrivia(trivia As SyntaxTrivia) As Boolean
             Return trivia.RawKind = SyntaxKind.ColonTrivia OrElse trivia.RawKind = SyntaxKind.LineContinuationTrivia
         End Function
 
-        Private Function ColonTriviaFollowedByLineContinuation(triviaList As IEnumerable(Of SyntaxTrivia)) As Boolean
-            Dim previousTrivia As SyntaxTrivia = Nothing
-            For Each trivia In triviaList
-                If previousTrivia.RawKind = SyntaxKind.None OrElse trivia.RawKind = SyntaxKind.None Then
-                    Continue For
+        <PerformanceSensitive("https://github.com/dotnet/roslyn/issues/30819", AllowCaptures:=False, AllowImplicitBoxing:=False)>
+        Private Shared Function LastOrDefaultTrivia(triviaListPair As (SyntaxTriviaList, SyntaxTriviaList), predicate As Func(Of SyntaxTrivia, Boolean)) As SyntaxTrivia
```

Comment:
this change behavior slightly? but not sure whether it matters. also, not sure whether this still matter.. probably added since old VB did some magic on this specific case.

Reply:
> this change behavior slightly?

There should not be any behavior change. I don't see a reason why the behavior would change, but maybe I'm just missing it?

Reply:
the existing code seems searching thing from beginning to end (forward) and new code seems searching from the end to the beginning (backward), so when there is multiple match, 2 will find opposite one?

Reply:
but since multiple of ": _" in same line is illegal, probably not matter?

Reply:
Ah, this is an artifact of the way the diff was computed. An old method was removed in da78b3e, and then a new completely unrelated method was added later at the location where the old method was.

Reply:
+1